ddpui/ddpairbyte/airbyte_service.py

```diff
@@ -126,7 +127,13 @@ def get_workspaces():
     return res
 
 
-def get_workspace(workspace_id: str) -> dict:
+# Example Pydantic schema for workspace response
+class WorkspaceResponse(BaseModel):
+    workspaceId: str
+    name: str
```

Comment:
Same here, you should hit airbyte's api or look at the docs to see what a workspace api returns. It has more fields than these two